### PR TITLE
Song lyrics

### PR DESCRIPTION
**Pull request by abhimanyuthakre**
Get song lyrics using "lyrics song_name". User can specify more details like "lyrics song_name movie/album"

### CODE REVIEW

File: windows/activities.md [hunk 1]
```diff
@@ -94,6 +95,7 @@ The commands for the various functions are as follows:
 * "***Who are you/Identify Youself/What is your name***" for B.E.N.J.I. to introduce itself.
 * Face detection feature will automatically fetch image for first time user. From next time, it will predict the authenticity of user. If user is authenticated, then B.E.N.J.I. will start running. If user wants to add another user, he/she can use "***face*** username". By default, username is "admin". So, make sure, while adding a new user, use "username" other than "admin". B.E.N.J.I. will fetch a single image of the user and store it in "dataset" folder in "C" drive. 
 NOTE: Make sure that system is able to fetch a good image of yours(be in a good lighting conditions). You can also directly add image to "dataset" folder in "C" drive.
+* "***lyrics*** _SongName_" gives lyrics of the mentioned song. If required, more details can be provided after the _SongName_.
```
Comment (sara-02) [inline windows/activities.md]: Can you please specify the format of "more" details. eg: `abc movie:123` or `abc 123` or `abc album 123`

File: windows/benji.py [hunk 2]
```diff
@@ -84,6 +84,16 @@ def events(frame,put):
 		cam.release()
 		cv2.destroyAllWindows()
 	
+	#Get lyrics
+	elif link[0] == "lyrics":
+		link = '+'.join(link[1:])
+		link = link.replace('+',' ')
+		title = link[1:]
+		goog_search = "https://www.google.com/search?sclient=psy-ab&client=ubuntu&hs=k5b&channel=fs&biw=1366&bih=648&noj=1&q=" + title + "lyrics"
```
Comment (sara-02) [inline windows/benji.py]: Do we have a better way of finding the lyrics than scrapping google result? 
Also, client=ubuntu is not a constant parameter.

**Comment by sara-02** [comment]
@the-ethan-hunt What are your thoughts?

**Comment by abhimanyuthakre** [comment]
Made requested changes to the activities.md. Removed unnecessary parameters in query search.
@sara-02  I previously asked about this same feature and @the-ethan-hunt asked me to do something similar to Shazam. I tried using acrcloud api but could not fetch any results. One other thing I tried is showing lyrics in the GUI itself but the library had some issues(showed wrong results) when I tested the function. I used "lyricswiki" for it. So this seems other alternative wherein the lyrics are accurate.

**Comment by the-ethan-hunt** [comment]
@sara-02 we can work on this one right now. Once we get something better, we can replace it.
@abhimanyuthakre can we work on sound recognition for this in future? IMO, would seem a better concept for the feature. (just a thought)

**Pull request merged**